File: Nível 1/1397 - Jogo do Maior/main.py
while True:
    quantia = int(input("A: "))

    if quantia == 0: break

    jgdr1 = 0
    jgdr2 = 0

    for i in range(1, quantia + 1):
        x1, x2 = map(int, input("B: ").split(" "))

        if x1 > x2:
            jgdr1 += 1
        elif x2 > x1:
            jgdr2 += 1
    print(jgdr1, jgdr2)

File: Nível 1/1397 - Jogo do Maior/main.lua
repeat
    local quantia = tonumber(io.read())

    if quantia == 0 then
        break
    end

    local jgdr1 = 0
    local jgdr2 = 0

    for i = 1, quantia do
        local x1, x2 = io.read():match("(%S+)%s+(%S+)")

        x1 = tonumber(x1)
        x2 = tonumber(x2)

        if x1 > x2 then
            jgdr1 = jgdr1 + 1
        elseif x2 > x1 then
            jgdr2 = jgdr2 + 1
        end
    end

    print(string.format("%d %d", jgdr1, jgdr2))
until false